src/java/fr/paris/lutece/plugins/forms/util/FormsConstants.java

```diff
@@ -189,6 +189,8 @@ public final class FormsConstants
     public static final String MESSAGE_LOAD_BACKUP = "forms.message.view.loadBackUp";
     public static final String MESSAGE_SUMMARY_TITLE = "forms.summary.title";
     public static final String MESSAGE_ERROR_NO_STEP = "forms.error.form.noStep";
+    public static final String MESSAGE_ERROR_CRENEAUX = "forms.message.error.creneaux";
```

Comment:
all constants must be in english language,
please check all constants

Reply:
deleted because not used in addition to present French